using_images/xpaas_images/data_grid.adoc

```diff
@@ -25,6 +25,9 @@ functionality not specific to the JBoss Data Grid xPaaS image can be found in th
 https://access.redhat.com/documentation/en/red-hat-jboss-data-grid/[JBoss
 Data Grid documentation on the Red Hat Customer Portal].
 
+[NOTE]
+OpenShift Container Platform release 3.1 supports JBoss Data Grid release 6.6 and earlier. If you are using OpenShift release 3.2 and JBoss Data Grid release 7.1 or later, refer to the documentation in the Red Hat Customer Portal.
+
```

Comment:
A few notes:

We markup admonitions with four =s. See https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions for more info.

It would be better to link to a specific place in the docs, but are you meaning that they should check in the OCP 3.2 docs?

Reply:
Thanks @oraNod . I didn't see that. This topic might have been made before we made that rule. Thanks for fixing it.